.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,112 @@
+name: Bug report
+description: ' Use this template if you''re running into bugs or other issues '
+labels: bug
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        ## Instructions
+
+
+        This is a template for the issue you are about to open! It will help you
+        provide the information necessary to complete your **bug report**.
+        Before you open an issue, ensure you've completed every step on the
+        checklist below.
+
+
+        - Have you used the [search
+        tool](https://github.com/CaffeineMC/sodium-fabric/issues) to find
+        similar issues? If you find one that looks like what you are getting,
+        consider contributing additional information to it instead. Make sure
+        you are not opening a duplicate.
+
+        - Are you using the latest version of the mod? If not, try updating to
+        see if it resolves your issue. You can find the latest versions of
+        CaffeineMC mods [here](https://caffeinemc.net).
+
+
+        All good? Then continue to fill the issue by providing the following
+        information about it:
+  - type: input
+    id: version
+    attributes:
+      label: Version information
+      description: >-
+        Please provide the exact version of the mod you are using. Every part of
+        the version is important! If you do not know what version you are using,
+        look at the file name in your `mods` folder.
+      placeholder: 'Example: mc1.16.4-0.1.1+rev.4b3964a'
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Behavior
+      description: What did you expect to happen?
+      placeholder: 'Example: The Piston does not extend.'
```

Comment:
Probably should be 'Example: The Piston extends'

Reply:
Fixed by changing it to the previous example.